lib/pause.js

```diff
@@ -125,4 +136,20 @@ function completer(line) {
   return [hits && hits.length ? hits : completions, line];
 }
 
+const setPassedVariables = (passedObject) => {
+  const registeredVariables = [];
+  if (passedObject) {
+    for (const key of Object.keys(passedObject)) {
+      registeredVariables.push(key);
+      if (typeof passedObject[key] === 'string') {
+        eval(`${key} = '${passedObject[key]}'`); // eslint-disable-line no-eval
+      } else {
+        eval(`${key} = ${passedObject[key]}`); // eslint-disable-line no-eval
```

Comment:
Hey there @DavertMik — I'm one of the users who requested this feature. I tried it on out master and have run into an issue when trying to pass a deeply nested object to the pause method e.g.
```javascript
{
  users: {
    admin: {
      email: '61abf3c910-16b952+uitest-admin@inbox.mailtrap.io',
      password: 'UIT3sts4Ever@#$',
      firstName: 'Admin',
      lastName: 'Insights'
    },
    tm: {
      email: '61abf3c910-16b952+uitest-tm@inbox.mailtrap.io',
      password: 'UIT3sts4Ever@#$',
      firstName: 'TM',
      lastName: 'Insights'
    }
  },
  fields: {},
  controls: {},
  urls: {}
}
```
I was able to fix the problem by wrapping `passedObject[key]` in a `JSON.stringify`.  I don't have time to fork / open a PR this week, but if it's still an issue I can open one next week.

Reply:
Thanks for testing, I will add this myself

Reply:
Awesome, thank you for responding to the feedback so quickly!